DQM/SiStripMonitorCluster/src/SiStripMonitorCluster.cc

```diff
@@ -732,227 +825,232 @@ void SiStripMonitorCluster::analyze(const edm::Event& iEvent, const edm::EventSe
+          long long tbx_corr = tbx;
+
+
+          if (SubDetPhasePartMap.find(sdet) != SubDetPhasePartMap.end()) the_phase = apv_phase_collection->getPhase(SubDetPhasePartMap[sdet]);
+          if(the_phase==APVCyclePhaseCollection::nopartition ||
+            the_phase==APVCyclePhaseCollection::multiphase ||
+            the_phase==APVCyclePhaseCollection::invalid) {
+              the_phase=30;
+              //std::cout << " subdet " << it->first << " not valid" << " MR " << MultiplicityRegion <<std::endl;
+            }
+            tbx_corr  -= the_phase;
+
+            long long dbxincycle = event_history->deltaBXinCycle(the_phase);
+            //std::cout<<"dbx in cycle: " << dbxincycle << std::endl;
+            if (globalswitchapvcycledbxth2on && !global_histo_filled) {
+              GlobalApvCycleDBxTH2->Fill(tbx_corr%70,dbx);
+              GlobalDBxTH1->Fill(dbx);
+              GlobalDBxCycleTH1->Fill(dbxincycle);
+              global_histo_filled = true;
+            }
+
+
+            // Fill MainDiagonalPosition plots ***************
+            if (cluster_detsetvektor_pix.isValid()){
+              if (ClusterHisto_){
+                if ( passBPTXfilter_ and passStripDCSfilter_ ){
+                  NumberOfStripClus_vs_BX->Fill(tbx%3564,NStripClusters);
+                  if(passPixelDCSfilter_ ){
+                    NumberOfPixelClus_vs_BX->Fill(tbx%3564,NPixClusters);
+                    if (globalswitchmaindiagonalposition && NStripClusters > 0){
+                      GlobalMainDiagonalPosition_vs_BX->Fill(tbx%3564,atan(NPixClusters/(k0*NStripClusters)));
+                      GlobalTH2MainDiagonalPosition_vs_BX->Fill(tbx%3564,atan(NPixClusters/(k0*NStripClusters)));
+                    }
+                  }
+                }
+                //Filling # clusters per FED ID histogram from FED Id clusters map (for all layers simultaneously).
+                map<int,int>::iterator it;
+                for(it=FEDID_v_clustersum.begin(); it!=FEDID_v_clustersum.end(); it++){
+                  NumberOfFEDClus->Fill(it->first,it->second);
+                }
+                FEDID_v_clustersum.clear();
+              }
+            }
+
+            if (isPixValid && !MultiplicityRegion_Vs_APVcycle_filled){
+              if (globalswitchstripnoise2apvcycle && MultiplicityRegion==2) {StripNoise2Cycle->Fill(tbx_corr%70);}
+              if (globalswitchstripnoise3apvcycle && MultiplicityRegion==3) {StripNoise3Cycle->Fill(tbx_corr%70);}
+              MultiplicityRegion_Vs_APVcycle_filled=true;
+            }
+
+            if (subdetswitchtotclusth1on)
+            sdetmes.SubDetTotClusterTH1->Fill(sdetmes.totNClusters);
+            if (subdetswitchtotclusprofon)
+            sdetmes.SubDetTotClusterProf->Fill(trendVar,sdetmes.totNClusters);
+            if (subdetswitchapvcycleprofon)
+            sdetmes.SubDetClusterApvProf->Fill(tbx_corr%70,sdetmes.totNClusters);
```

Comment:
Something odd happening with the file change history here as this is not code implemented by me.